### DISCUSSION TITLE
Keep the architecture view

### DISCUSSION THREAD

**Discussion by PterPmnta** [ArchSense/archsense-mono#12] [Ideas]
The plugin is really awesome but would be more incredible, if it were possible to keep the architecture view, once is reorganized the structure.

**Comment by bolshchikov** [comment]
@PterPmnta , agree but that would require some sort of persistency in VSCode. 
I'm working in a web-based version that will allow to connect to github and do a diff between commits.
It will have persistency like you described.

**Comment by bolshchikov** [comment]
@PterPmnta , created an issue for it #22.
